airflow/www/views.py

```diff
@@ -2142,7 +2142,7 @@ def get_int_arg(value, default=0):
             query = query.filter(~DM.is_paused)
 
         if arg_search_query:
-            query = query.filter(sqla.func.lower(DM.dag_id) == arg_search_query.lower())
+            query = query.filter(DagModel.dag_id.ilike('%' + arg_search_query + '%'))
```

Comment:
To do true token based subsequence search I think you should do:
```suggestion
            query = query.filter(DagModel.dag_id.ilike('%' + arg_search_query.replace(' ', '%') + '%'))
``

Reply:
to make sure i understand, this would match by the given ordered space-separated tokens, correct? so "ello orl" would match "hello to the world" for example?

Reply:
yes that is correct!